### PR TITLE
Set dummy loglevel to debug

### PR DESCRIPTION
**Pull request by maaktweluit**
This way when they fail we can investigate more

### CODE REVIEW

**Review by mfranciszkiewicz — DISMISSED** [review]

File: loggingconfig.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
         'console': {
             'class': 'logging.StreamHandler',
             'level': 'INFO',
-            'formatter': 'simple',
+            'formatter': 'date',
```
Comment (tworec) [inline loggingconfig.py]: this will affect whole app, not only dummy task

**Review by tworec — CHANGES_REQUESTED** [review]

File: loggingconfig.py (hunk 1 above)
Comment (maaktweluit) [reply loggingconfig.py]: Good one!

This is not meant to be merged, this was required to find what component is blocking

Changed title to WIP, will ping you when its ready for review again

**Comment by maaktweluit** [comment]
Turned out UPNP was the problem this time...

Left some logs and added the dates in this test only.

Hope the next time dummy tests failures are easier to spot :)

@mfranciszkiewicz @tworec please review again!

**Review by mfranciszkiewicz — APPROVED** [review]

File: tests/golem/task/dummy/runner.py [hunk 2]
```diff
@@ -61,6 +68,7 @@ def create_client(datadir):
     config_desc = ClientConfigDescriptor()
     config_desc.init_from_app_config(app_config)
     config_desc.key_difficulty = 0
+    config_desc.use_upnp = False
```
Comment (tworec) [inline tests/golem/task/dummy/runner.py:71]: seems it's only line changed not regarding logger. Is this cause of our headache? :)

**Review by tworec — APPROVED** [review]

File: tests/golem/task/dummy/runner.py (hunk 2 above)
Comment (maaktweluit) [reply tests/golem/task/dummy/runner.py:71]: Yep, the one and only...

Hopefully the rest makes it easier to find next time

**Pull request merged**